src/H5Dcontig.c

```diff
@@ -658,7 +658,7 @@ H5D__contig_read(H5D_io_info_t *io_info, const H5D_type_info_t *type_info, hsize
         /* Issue selection I/O call (we can skip the page buffer because we've
          * already verified it won't be used, and the metadata accumulator
          * because this is raw data) Only call funciton if nelmts > 0. */
```

Comment:
Does this header comment need updated in relation to now always calling the function below?

Reply:
Oops, thanks, I'll fix that when I run the formatter.